setup.py

```diff
@@ -224,7 +224,8 @@ def get_data_files():
     'pandocfilters>=1.4.1',
     'testpath',
     'defusedxml',
-    'nbclient>=0.5.0,<0.6.0'
+    'nbclient>=0.5.0,<0.6.0',
+    'ipython_genutils',
```

Comment:
```suggestion
    'ipython-genutils',
```

Could write it this way instead.